.prettierignore

```diff
@@ -1,12 +1,12 @@
-
-node_modules
-package-lock.json
-
-operators.md
-dataframes.md
-nushell_map.md
-*_bash*.md
-book/commands/*.md
-.vuepress/.temp
-.vuepress/.cache
-.vuepress/dist/
+

```

Comment:
Did you mean to update this file? The GitHub web UI doesn't make it obvious what changed (probably line endings?)

Reply:
Just line endings… I think. I didn't touch it myself.